pages/device/[id]/device.js

```diff
@@ -9,6 +9,7 @@ import {
   WorkIcon,
   LightingIcon,
```

Comment:
If you name this file `device.js` the route of your details page is `/device/<id>/device`. I think you want to have a path like this: `/device/id`. Then you should rename this file as `index.js`. :)

Reply:
Yeah, the path name is not the best, but most definitely dislike it more, if I have more than one index.js. I get lost in my files if I do that 😞. Do I have to change it?

Reply:
done :)